modules/exploits/windows/http/file_sharing_wizard_seh.rb

```diff
@@ -0,0 +1,57 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super update_info(info,
+                      'Name' => 'File Sharing Wizard 1.5.0 - POST SEH Overflow',
```

Comment:
We generally leave the version numbers in the description and/or targets.

Reply:
will change

Reply:
It's just that one module may support multiple versions, and you'd note that in the description or as individual targets.